### ISSUE TITLE
No continuous arrangement of modules

### ISSUE THREAD

**Issue by Arnaud73** [militu/electrical-panel-label#4]
My main electric enclosure contains many modules, most of them are arranged starting from the left. However, I have a few that are arranged starting from the right, with empty spaces in-between. Those modules can not placed without inserting fake modules.

**Comment by militu** [comment]
Yes you are right, the use of fake modules (or empty ones let's say) is the logic to maintain correct alignement. You can easily add a large number of empty modules in one go using the "add modules" with the number of elements. Currently I don't see any easy way to allow complete flexibility without using those "empty modules". Do you have any suggestions for this?
